cmd/oras/copy.go

```diff
@@ -143,6 +157,20 @@ func runCopy(opts copyOptions) error {
 	}
 
 	fmt.Println("Copied", opts.srcRef, "=>", opts.dstRef)
+
+	if len(opts.extraRefs) != 0 {
+		var contentBytes []byte
+		fetchOpts := oras.DefaultFetchBytesOptions
+		fetchOpts.TargetPlatform = targetPlatform
+		if _, contentBytes, err = oras.FetchBytes(ctx, src, opts.srcRef, fetchOpts); err != nil {
+			return err
+		}
+
+		tagBytesNOpts := oras.DefaultTagBytesNOptions
+		tagBytesNOpts.Concurrency = opts.concurrency
+		oras.TagBytesN(ctx, &display.TagManifestStatusPrinter{Repository: dst}, desc.MediaType, contentBytes, opts.extraRefs, tagBytesNOpts)
```

Comment:
Since it is remote to remote copy, should we just use `oras.TagN()`?

Reply:
@wju-MSFT Here we can use `TagN` because we don't have a descriptor like `oras push` to help saving one resolve request.

Reply:
done